spiketools/tests/plts/test_neuron.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for spiketools.plts.neuron"""
```

Comment:
Same comment as naming in sim, this file name is too vague, since it's scope is not "visualizing neurons", but rather "visualizing place cells". Name needs updating